README.md

```diff
@@ -1,29 +1,45 @@
 # web3-login-demo
 
-This website demonstrates how to build a logging using KILT Protocol.
+This project demonstrates how to enable **login using KILT Protocol**. 
+It is an educational project, everything is broken down to little steps and has explanations to ease your understanding. 
 
-During the logging procedure, the user is required to present a credential.
+During the login procedure, the user is required to present a credential.
 You can decide which credentials to accept.
 They could be issued by yourself or by other attesters.
-In this demo we request a credential that contains an email address that the user owns.
-For that we rely on https://socialkyc.io to issue email credentials.
+
+This is basically like asking for an ID-Card instead of a customer number, in order to provide a personalized service.
+This means that users don't not need to setup an account and password just for your website.
+Additionally, this avoids third parties (usually powerfull data-collecting companies) from tracking your interactions with your clients.
+Not even the KILT developers can track it. 
+
 
 In order for a **dApp** to support logging in with KILT Credentials, it needs:
 
-1. An on-chain DID
+1. It's on-chain DID
+
+- This DID is used so that the user knows the parties to whom they talk.
+
+2. It's [Domain Linkage Credential](#Well-Known-DID-Configuration)
+
+- Bind your DID to a specific domain.
+- This prevents Man-in-the-Middle attacks.
+- Also known as the _well-known-did-configuration_.
 
-- This DID is used so that the user knows the parties to whom they talk
+3. A CType to request from the user
+- The type of credentials the dApp considers valid. 
+- In this demo, as a default, we request a credential that contains an email address.
+For that we rely on [SocialKYC](https://socialkyc.io) to issue email credentials after verifing the user owns it.
+You could easily modified this. 
 
-2. Domain linkage credential
 
-- Bind your DID to a specific domain
-- This prevents Man-in-the-Middle attacks
+If you don't have some of the above, don't worry, we help you get them on the sections below. 
 
-3. A CType you want to request from the user
+After setting up and running this project locally on your computer, you will know how the [login process](#Process) works and have code that you can implement on your website. 
+We encourage you to customize the code for your specific usecase, just keep in mind the [specifications](https://github.com/KILTprotocol/spec-ext-credential-api) to retain compatibility. 
 
-- In this example, we use the email CType, but you could use any CType you want
+If you want to implement KILT Login without understanding it, we also have a solution for you. 
+There is a [containerized version under this repository](https://github.com/KILTprotocol/kilt-login). 
```

Comment:
```suggestion
As mentioned, this is an educational project and is not suitable for production. There is a production-ready solution that can be easily integrated into your infrastructure.
```

Reply:
This is also suitable for production. There are Dockerfiles and everything. There is a deployed version. 

The difference with the containerised version is that you would actually integrated with your website instead of having a middle man doing the work for you. 

Your website would actually work with KILT Credentials instead of just a JWT.

Reply:
Maybe you like it better now.